simoc_server/agent_model/agents/structure.py

```diff
@@ -402,46 +404,53 @@ def ship(self, edible, inedible):
 
 class Planter(EnclosedAgent):
     agent_type_name = "planter"
-    # TODO right now just grows generic plant, the planter should choose a specific type somehow
     # TODO planter plants everything in one step, should be incremental
     # TODO planter should use soil
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.counter = 0
```

Comment:
counter is not persisted, if the model is saved and loaded it will not behave deterministically